plugins/odf/console-extensions.json

```
@@ -75,6 +75,45 @@
       "required": ["ODF_MODEL", "ODF_ADMIN"]
     }
   },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "storage-overview",
+      "insertBefore": "odfdashboard",
+      "section": "storage",
+      "name": "%plugin__odf-console~Overview%",
+      "href": "/odf/overview"
+    },
+    "flags": {
+      "required": ["ODF_MODEL"]
+    }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "storage-cluster",
+      "insertAfter": "storage-overview",
+      "section": "storage",
+      "name": "%plugin__odf-console~Storage Cluster%",
+      "href": "/odf/storage-cluster"
+    },
+    "flags": {
+      "required": ["ODF_MODEL", "ODF_ADMIN"]
+    }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "storage-object",
+      "insertAfter": "storage-overview",
+      "section": "storage",
+      "name": "%plugin__odf-console~Obejct Storage%",
+      "href": "/odf/mcg"
```

```suggestion
      "href": "/odf/object-storage"
```
we have rgw as well, `/mcg` seems incorrect...